drivers/dai/virtual/virtual_dai.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2025 Suraj Sonawane <surajsonawane0215@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/drivers/dai.h>
+#include <zephyr/pm/device_runtime.h>
+#include <string.h>
+
+#define DT_DRV_COMPAT virtual_dai
+LOG_MODULE_REGISTER(virtual_dai, CONFIG_DAI_LOG_LEVEL);
+
+struct virtual_dai_data {
+	struct dai_config cfg;
+};
+
+static int virtual_dai_probe(const struct device *dev)
+{
+	LOG_INF("%s", __func__);
```

Comment:
I'd reduce the number of these LOG_INF()s in the merged version. I know this virtual DAI is more for R&D use, so logging can be on the more verbose side, but I'd still drop the function entry/exit logs (especially as Zephyr logging can add the function info with a build option, no need to add "__func__" in driver code.